webboot/webboot.go

```diff
@@ -79,18 +79,13 @@ func linkOpen(URL string) (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
-// write copies the content from an io.readCloser to a named file with filename.
-func write(read io.ReadCloser, filename string) error {
-	w, err := os.Create(filename)
-	if err != nil {
-		return err
-	}
-
-	defer w.Close()
-	defer read.Close()
+// TODO: Find a persistent memory block large enough to store the content. If no blocks are available, error the user.
+// write copies the content from an io.readCloser to a persistent memory block.
+func write(body []byte, pmem *os.File) error {
```

Comment:
you can remove this function completely.

Reply:
I was thinking the same. Using the go packages for opening the file, and copying the contents over to another file eliminates having to use an entire function.